lib/containers/utils.pm

```diff
@@ -79,6 +79,49 @@ sub registry_url {
     return sprintf("%s/%s:%s", $repo, $container_name, $version_tag);
 }
 
+sub test_update_cmd {
+    my %args = @_;
+    my $runtime = $args{runtime};
+    my $container = $args{container};
+
+    # podman update was added to v4.3.0
+    if ($runtime eq 'podman') {
+        my $version = script_output "podman version | awk '/^Version:/ { print \$2 }'";
+        if (package_version_cmp($version, '4.3.0') <= 0) {
+            record_info("The update command is not supported on podman $version");
+            return;
+        }
```

Comment:
What about instead of this check we check for current SLE / Leap version?

Reply:
> What about instead of this check we check for current SLE / Leap version?

Leap 15.4 & 15.5 still use 3.4.7.  Only Tumbleweed has 4.3.1.  We can change this later when >4.3 lands in Leap.

Reply:
OK. Let's keep it as it is.